### ISSUE TITLE
Cesium crashes if a layer has defined credits without defining text or title

### ISSUE THREAD

**Issue by MV88** [geosolutions-it/MapStore2#8910]
## Description

App crash when a WMS layer has credits defined badly, with no text or title

which make cesium 1.90 crash here
```
function Credit(html, showOnScreen) {
  //>>includeStart('debug', pragmas.debug);
  Check.typeOf.string("html", html); <--<--<--<--<-- HERE html is undefined
  //>>includeEnd('debug');
  let id;
  const key = html;

  if (defined(creditToId[key])) {
```

check if other layers in cesium have the same problem,
like WMTS

## How to reproduce

- try viewing this map in cesium
[cesiumCrash.txt](https://github.com/geosolutions-it/MapStore2/files/10411240/cesiumCrash.txt)

```json
{
  "version": 2,
  "map":{
      "projection":"EPSG:900913",
      "units":"m",
      "center":{
         "x":1250000.000000,
         "y":5370000.000000,
         "crs":"EPSG:900913"
      },
      "zoom":5,
      "maxExtent":[
         -20037508.34,
         -20037508.34,
         20037508.34,
         20037508.34
      ],
      "layers":[
        {
          "format":"image/jpeg",
          "group":"background",
          "name":"s2cloudless:s2cloudless",
          "opacity":1,
          "title":"Sentinel 2 Cloudless",
          "type":"wms",
          "credits" : {
            "imageUrl": ""
          },
          "url":[
             "https://1maps.geo-solutions.it/geoserver/wms",
             "https://2maps.geo-solutions.it/geoserver/wms",
             "https://3maps.geo-solutions.it/geoserver/wms",
             "https://4maps.geo-solutions.it/geoserver/wms",
             "https://5maps.geo-solutions.it/geoserver/wms",
             "https://6maps.geo-solutions.it/geoserver/wms"
          ],
          "source":"s2cloudless",
          "visibility":true,
          "singleTile":false
       }
      ]
   }
}
```

*Expected Result*

to view the map

*Current Result*

crash with this error

```
"Error
    at new DeveloperError (webpack://MapStore2/./node_modules/cesium/Source/Core/DeveloperError.js?:44:11)
    at Check.typeOf.string (webpack://MapStore2/./node_modules/cesium/Source/Core/Check.js?:74:11)
    at new Credit (webpack://MapStore2/./node_modules/cesium/Source/Core/Credit.js?:33:57)
    at wmsToCesiumOptions (webpack://MapStore2/./MapStore2/web/client/components/map/cesium/plugins/WMSLayer.js?:77:21)
    at Object.createLayer [as create] (webpack://MapStore2/./MapStore2/web/client/components/map/cesium/plugins/WMSLayer.js?:115:82)
    at Object.createLayer (webpack://MapStore2/./MapStore2/web/client/utils/cesium/Layers.js?:21:27)
    at CesiumLayer.eval [as createLayer] (webpack://MapStore2/./MapStore2/web/client/components/map/cesium/Layer.jsx?:148:81)
    at CesiumLayer.componentDidMount (webpack://MapStore2/./MapStore2/web/client/components/map/cesium/Layer.jsx?:223:12)
    at commitLifeCycles (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:21964:22)
    at commitLayoutEffects (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:25288:7)
    at HTMLUnknownElement.callCallback (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:362:14)
    at Object.invokeGuardedCallbackDev (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:411:16)
    at invokeGuardedCallback (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:466:31)
    at commitRootImpl (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:25024:9)
    at unstable_runWithPriority (webpack://MapStore2/./node_modules/scheduler/cjs/scheduler.development.js?:815:12)
    at runWithPriority$2 (webpack://MapStore2/./node_modules/react-dom/cjs/react-dom.development.js?:12188:10)"
```

- [x] Not browser related

<details><summary> <b>Browser info</b> </summary>

(use this site: <a href="https://www.whatsmybrowser.org/">https://www.whatsmybrowser.org/</a> for non expert users)

| Browser Affected | Version |
|---|---|
|Internet Explorer| |
|Edge| |
|Chrome| |
|Firefox| |
|Safari| |
</details>

**Comment by tdipisa** [comment]
@MV88 at a really first look this seems a duplication or at least connected with this https://github.com/geosolutions-it/MapStore2/issues/8894. Can you confirm @MV88 @offtherailz?

**Comment by MV88** [comment]
it crash also with other type of layers, so i guess we can make a generic issue and merge the two

**Comment by tdipisa** [comment]
@MV88 can you take care of it please?
Thank you so much

**Comment by MV88** [comment]
closed this in favor of #8894
